recipe/meta.yaml

```
@@ -1,46 +1,59 @@
 {% set name = "gguf" %}
-{% set version = "0.6.0" %}
+{% set version = "0.9.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gguf-{{ version }}.tar.gz
-  sha256: b2e22eaba2a106c1ad148186a14ad9f29c429351686fe9d7ce16df39a0607e65
+  sha256: f5e70987e15b19c545f6a9f7533b4033fb306330eb1f3c5c95fd28b14c3fd33b
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
   entry_points:
-    - gguf-convert-endian = "scripts:gguf_convert_endian_entrypoint"
-    - gguf-dump = "scripts:gguf_dump_entrypoint"
-    - gguf-set-metadata = "scripts:gguf_set_metadata_entrypoint"
+    - gguf-convert-endian = scripts:gguf_convert_endian_entrypoint
+    - gguf-dump = scripts:gguf_dump_entrypoint
+    - gguf-set-metadata = scripts:gguf_set_metadata_entrypoint
+    - gguf-new-metadata = scripts:gguf_new_metadata_entrypoint
+  skip: True # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
 
 requirements:
   host:
-    - python >=3.8
+    - python
+    - poetry-core >=1.0.0
     - pip
-    - poetry-core
   run:
-    - python >=3.8
+    - python
     - numpy >=1.17
+    - tqdm >=4.27
+    - sentencepiece >=0.2.0 # This dependency is missing in the upstream pyproject.toml file: https://github.com/ggerganov/llama.cpp/blob/master/gguf-py/pyproject.toml
```

```suggestion
    - pyyaml >=5.1
    - sentencepiece >=0.2.0 # This dependency is missing in the upstream pyproject.toml file: https://github.com/ggerganov/llama.cpp/blob/master/gguf-py/pyproject.toml
```

According to this line: https://github.com/ggerganov/llama.cpp/blob/2f3c1466ff46a2413b0e363a5005c46538186ee6/gguf-py/pyproject.toml#L25

---

It would seem the correct `pyyproject.toml` is [this](https://github.com/ggerganov/llama.cpp/blob/gguf-v0.9.1/gguf-py/pyproject.toml) one and there doesn't seem to be a mention of `pyyaml`.

---

Thank you @ryanskeith and @Jrice1317 - I just caught that too, they're missing pyyaml but not sure how my build passed missing it. I also just realized there's an issue linking / using the pypi package version vs the upstream source. 

I'm going to commit a fix that moves to the upstream source (and add pyyaml) instead of the published pypi package and use a version scheme similar to llama.cpp-feedstock.

---

What the problem with the version @jnoller? https://inspector.pypi.io/project/gguf/0.9.1/packages/9f/1e/6bd024e0138d663cd333e8fde4c03d343e8be680d43b86537ef1497a7e32/gguf-0.9.1.tar.gz/gguf-0.9.1/pyproject.toml matches  https://github.com/ggerganov/llama.cpp/blob/gguf-v0.9.1/gguf-py/pyproject.toml from what I see...